admin_manual/release_notes.rst

```diff
@@ -12,6 +13,61 @@ Release Notes
 * :ref:`8.0_release_notes_label`
 * :ref:`7.0_release_notes_label`
 
+.. _10.0.4_release_notes_label:
+
+Changes in 10.0.4
+-----------------
+
+Dear ownCloud administrator, please find below the changes and known issues of ownCloud Server 10.0.4 that need your attention:
+
+**The full ownCloud Server 10.0.4 changelog can be found here: https://github.com/owncloud/core/blob/stable10/CHANGELOG.md**
+
+* **More granular sharing restrictions**
+The option "Restrict users to only share with users in their groups" in the admin sharing settings forces users to only be able to share with groups they are member of while simultaneously prohibiting to share with single users that do not belong to any of the users' groups. To make this more granular we split this option into two and added "Restrict users to only share with groups they are member of" which now differentiates between users and groups making it possible to force users to be able to share with all users of an installation but only with groups they are member of and vice versa. Please check if the new possibilities better suit your needs.
```

Comment:
s/they are member/they are a member/g

Reply:
Note: the text on the UI says "...they are member of"
one day, maybe, that could be changed to say "...they are a member of"

Reply:
> Please check if the new possibilities better suit your needs.

Is imho not necessary, as the old way still exists and the new implementation is an addition

Reply:
thanks, changed. as @phil-davis said the option is called like this in the UI..